[BUG] ESPAsyncWebServer.h question

Running newest versions, Windows/ArduinoIDE 232+ 30Jul24 nightly. (v313)
line157 change error:

Documents\Arduino\libraries\ESPAsyncWebServer\src/ESPAsyncWebServer.h: In member function 'String AsyncWebHeader::toString() const':
c:\Users\jelli\Documents\Arduino\libraries\ESPAsyncWebServer\src/ESPAsyncWebServer.h:157:44: error: ambiguous overload for 'operator+' (operand types are 'const String' and 'char')
  157 |     String toString() const { return _name + (char)0x3a + (char)0x20 /*": "*/ + _value + asyncsrv::T_rn; }
      |                                      ~~~~~ ^ ~~~~~~~~~~
      |                                      |       |
      |                                      |       char
      |                                      const String

Thank you for any feedback.

--- a/src/ESPAsyncWebServer.h
+++ b/src/ESPAsyncWebServer.h
@@ -154,7 +154,14 @@ class AsyncWebHeader {
 
     const String& name() const { return _name; }
     const String& value() const { return _value; }
-    String toString() const { return _name + (char)0x3a + (char)0x20 /*": "*/ + _value + asyncsrv::T_rn; }
+    String toString() const {
+      String str = _name;
+      str.concat((char)0x3a);
+      str.concat((char)0x20);
+      str.concat(_value);
+      str.concat(asyncsrv::T_rn);
+      return str;
+    }
 };
 
 /*

